iac-extensions/kubernetes/src/test/resources/checks/CpuLimitCheck/test_template_object.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         # ^^^^
           resources:
             requests:
-              memory: 64Mi
+              cpu: 64Mi
```

Comment:
Why is this changed? Is `Mi` a valid quantifier for CPU?

Reply:
Good catch, the `Mi` should be removed. I changed it here, because I remarked this on another PR and it was missed to change.